recipes/csvlens/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "csvlens" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/YS-L/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0f8b14f929c5acdc697187ba5c5a1ae643f97dcfc9325847c60f78cb73a6328c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("rust") }}
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+    - cargo-bundle-licenses
+
+test:
+  commands:
+    - csvlens --version
+
+about:
+  home: https://github.com/YS-L/csvlens
+  dev_url: https://github.com/YS-L/csvlens
+  license: MIT
+  license_file: LICENSE
```

Comment:
You generated the `THIRDPARTY.yml` license file, but aren't including it:
https://conda-forge.org/docs/maintainer/adding_pkgs/#rust

Reply:
Done!